zambeze/cli_agent.py

```diff
@@ -42,24 +39,16 @@ def start(
         agent_logger.setLevel(logging.INFO)
     agent_logger.addHandler(fh)
 
-    agent_logger.info("*****************************************************")
     agent_logger.info("*****************************************************")
     agent_logger.info("Creating Zambeze agent subprocess with configuration:")
     agent_logger.info("*****************************************************")
     agent_logger.info(f"Log Path:\t\t{log_path}")
     agent_logger.info(f"Config Path:\t\t{config_path}")
     agent_logger.info(f"Debug Logs:\t\t{debug}")
-    agent_logger.info(f"ZMQ activity port:\t{zmq_activity_port}")
-    agent_logger.info(f"ZMQ heartbeat port:\t{zmq_heartbeat_port}")
-    agent_logger.info("*****************************************************")
     agent_logger.info("*****************************************************")
 
     # Quick fix to address empty string in CL-args.
```

Comment:
Do we still need this block with argparse?

Reply:
sorry, I mis-highlighted. I'm talking about lines 57-62.

Reply:
Based on the discussion in Teams, the CLI is going to be redone with argparse and the commands simplified. We can leave this for now because the next merge request will make these changes.